src/app/login/login.component.ts

```diff
@@ -13,7 +13,7 @@ export class LoginComponent {
     password: new FormControl('', Validators.required),
   });
 
-  constructor(private loginService: LoginApiService) {}
+  constructor(private loginService: AuthenticationService) {}
```

Comment:
This is still called loginService which is wrong since we are injecting authenticationService

it should be `private authService: AuthenticationService` for example